client/server/user-bootstrap/index.js

```diff
@@ -20,11 +18,6 @@ const SUPPORT_SESSION_COOKIE_NAME = 'support_session_id';
  * WordPress.com REST API /me endpoint.
  */
 const API_PATH = 'https://public-api.wordpress.com/rest/v1/me';
-const apiQuery = {
-	meta: 'flags',
-	abtests: getActiveTestNames( { appendDatestamp: true, asCSV: true } ),
-};
-const url = `${ API_PATH }?${ stringify( apiQuery ) }`;
```

Comment:
The `meta: 'flags'` query param needs to stay here, too. Removing `stringify` is a good idea: could be replaced either by `addQueryArgs` or by the native `URLSearchParams`.

Reply:
Good call. Refactored to use `URLSearchParams`.